### ISSUE TITLE
Liberapay

### ISSUE THREAD

**Issue by rugk** [microg/android_packages_apps_GmsCore#508]
Seeing you now have a donation button, I wanted to suggest another donation method. https://liberapay.com/

It is easy to use and the advantage is you get reoccurring donations. One can pay via bank transfer etc.
And LiberaPay is especially made for FLOSS projects, itself being a non-profit.

(moved from https://github.com/lineageos4microg/l4m_website/issues/10)

**Comment by mar-v-in** [comment]
I already set up liberapay at https://liberapay.com/microG/

However, I got the feeling that many people like to do one time donations, which is not possible with liberapay. Additionally bountysource has the issue bounty feature, which some like to use to express interest in a certain feature (there are 440 US-$ in issue bounties in bountysource).

Longterm, I'd like to phase out bountysource (especially since they were bought by this shady ICO blockchain thing), but that's not something to be done in just a few days or weeks.
